src/fsharp/PackageManager.Integration.fs

```diff
@@ -38,15 +38,17 @@ type PackageManager = {
     ToolName: string
     Name:string }
 
-let RegisteredPackageManagers : PackageManager list = 
+let RegisteredPackageManagers = lazy (
```

Comment:
or if VF# doen't want to deploy the paket.packageloader.dll then we still should have that folder where things are automatically loaded. Then paket user can install it there themselves